dictionary.txt

```diff
@@ -0,0 +1,155 @@
+.gitignore
+100MB
+10MB
+256kb
+2FA
+300ms
+3MB
```

Comment:
Would it be possible to check for these sorts of number + unit keywords and ignore them generally?

Reply:
I'm working on adding a command-line option that takes a list of regexes. If a spelling mistake matches any of the regexes (after wrapping each regex in `^` and `$`), the spelling mistake is ignored. For example, to ignore the megabyte amounts currently in the dictionary, `--ignore \d+MB` would work I think. That feature is in `master` but I haven't published it to `npm` yet. 

I'm wondering if it would make more sense to make the command-line option take a file path, with the file containing the list of regexes.